webpack.config.dev.js

```diff
@@ -33,14 +34,16 @@ const devConfig = {
         test: /\.mjs$/,
         include: /node_modules/,
         type: 'javascript/auto',
+        resolve: {
```

Comment:
Technically, this resolve can be scoped to just `graphql` lib, but it seemed the cleanest to include here. It seems OK to leave as false.

https://webpack.js.org/configuration/module/#resolvefullyspecified